src/graphql/public/root/mutation/ln-usd-invoice-fee-probe.ts

```diff
@@ -72,7 +72,7 @@ const LnUsdInvoiceFeeProbeMutation = GT.Field<
       currencyId: "3", 
     })
     if (resp instanceof IbexClientError) return { errors: [mapAndParseErrorForGqlResponse(resp)] }     
-    if (!resp.amount) return { errors: [mapAndParseErrorForGqlResponse(new UnexpectedResponseError("Amount field not found."))] }
+    if (resp.amount === null || resp.amount === undefined) return { errors: [mapAndParseErrorForGqlResponse(new UnexpectedResponseError("Amount field not found."))] }
```

Comment:
any chance this can take values like NaN ? or empty string? line 70 should not be an "any" type in an ideal case. Approved, since I'm guessing it solves the problem for most cases.

Reply:
we could also do `if (!resp.amount && resp.amount !== 0)` which which would cover a broader number of cases i guess

Reply:
Agree, ideally the return type should not be "any". This seems to be a bug/typo in the Ibex SDK where the datatype is 'unknown'. So we have to treat it as 'any.' But this is expected to be number and it should be a required field, so really not too worried about these edge cases anyway